docs/eventing/samples/sequence/sequence-terminal/README.md

```diff
@@ -1,6 +1,7 @@
 We are going to create the following logical configuration. We create a
-CronJobSource, feeding events to a [`Sequence`](../../../flows/sequence.md). Sequence
-can then do either external work, or out of band create additional events.
+PingSource, feeding events to a [`Sequence`](../../../flows/sequence.md).
+Sequence can then do either external work, or out of band create additional
```

Comment:
This sentence may be difficult to understand for non native speakers. "Sequence can then either do external work, or create additional events out of band", maybe that will sound a little better but I'm not 100% sure that it means the same thing we're trying to say in this line.

Reply:
lol, probably because IIRC it was originally written by a non native speaker (I) :)

Reply:
IMHO it doesn't make sense to native speakers either - I really don't understand what this is saying?

Also, avoid using "we" please throughout docs. I see this in PRs still quite a bit, but it's not recommended, as per the style guide: https://developers.google.com/style/person